src/HttpCache/SouinPurger.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\HttpCache;
+
+use GuzzleHttp\ClientInterface;
```

Comment:
You should use this interface `\Symfony\Contracts\HttpClient\HttpClientInterface` instead